res/controllers/Keyboard-Default.kbd.xml

```diff
@@ -0,0 +1,382 @@
+<?xml version="1.0" ?>
+<MixxxKeyboardPreset mixxxVersion="2.0.1+" schemaVersion="1">
+  <info>
+    <name>Default Mapping</name>
+    <author>Mixxx</author>
+    <description>Default mapping for Mixxx.</description>
+  </info>
+  <controller>
+    <group name="[AutoDJ]">
+      <control action="enabled">
+        <keyseq final="1">Shift+F12</keyseq>
+      </control>
+      <control action="fade_now">
+        <keyseq final="1">Shift+F11</keyseq>
+      </control>
+      <control action="shuffle_playlist">
+        <keyseq final="1">Shift+F9</keyseq>
+      </control>
+      <control action="skip_next">
+        <keyseq final="1">Shift+F10</keyseq>
+      </control>
+    </group>
+    <group name="[Channel1]">
+      <control action="LoadSelectedTrack">
+        <keyseq final="1">Shift+Left</keyseq>
+      </control>
+      <control action="back">
+        <keyseq lang="en_US" scancode="30">a</keyseq>
```

Comment:
> However, it looks like it exposes too much internals to the user.

True, although I don't know if that would still be an issue as soon as we have the mapping GUI.

> Do we need the scancode option?

I am a bit sceptical about only having the possibility to target keys on their keysyms _*_ rather than on their scancodes because there might be keys that share the same keysym. To find out if this was a thing, I scanned our current [layouts](https://github.com/Tomasito665/mixxx/blob/develop/src/controllers/keyboard/layouts.cpp) for these type of keys that share the same keysym, with the same modifier. This was the outcome:
```
On layout English (US), <AB09> is identical to <LSGT> with shift. Character: '>'
On layout Greek (Greece), <AC02> is identical to <AD02> with shift. Character: 'Σ'
```

Which means that we can not do:
```XML
<!-- In both cases there are two possible scancodes, based on our layouts data -->
<keyseq byPositionOf="en_US">Shift+&gt;</keyseq>
<keyseq byPositionOf="el_GR">Shift+Σ</keyseq>
```

However, these two are special cases, because:
 1. Keyboards for `en_US` English (US) layouts usually don't have this 105th <LSGT> key.
 2. Both keys <AC02> and <AD02> hold different lower-case sigma's and thus it's logical that they will produce the same upper-case sigma Σ when shifted.

So, I think we can live with it. Though, this does only say something about the 16 layouts we currently have in `layouts.cpp`. So, I don't know if this could potentially become a problem if we add more layouts, which do have more of these duplicate keys.

What do you think? Shall I just implement it?

_* We do not map on keysyms directly but on [their Unicode characters](https://github.com/Tomasito665/mixxx/blob/develop/scripts/layouts_tool/utils.h#L15)._